Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1886,19 +1897,12 @@ CGImageRef CGBitmapContextGetImage(CGContextRef context) {
     return ((__CGBitmapContext*)context)->Impl().image.get();
 }
 
-CGContextRef _CGBitmapContextCreateWithTexture(
-    int width, int height, float scale, DisplayTexture* texture, DisplayTextureLocking* locking) {
-    CGImageRef newImage = nullptr;
-    __CGSurfaceInfo surfaceInfo = _CGSurfaceInfoInit(width, height, _ColorARGB);
-    newImage = new CGGraphicBufferImage(surfaceInfo, texture, locking);
-
-    ComPtr<ID2D1RenderTarget> renderTarget = newImage->Backing()->GetRenderTarget();
-    renderTarget->SetDpi(96 * scale, 96 * scale);
-
+CGContextRef _CGBitmapContextCreateWithTexture(CGImageRef texture, float scale) {
     __CGBitmapContext* context = __CGBitmapContext::CreateInstance(kCFAllocatorDefault);
```

Comment:
Even cooler: QuartzCore can do this itself, and call `_CGContextCreateWithD2DRenderTarget` :)